packages/big-design/package.json

```diff
@@ -64,6 +65,7 @@
     "@types/jest": "^25.2.1",
     "@types/node": "^13.1.8",
     "@types/react": "^16.8.8",
+    "@types/react-datepicker": "^2.11.0",
```

Comment:
Since we are using and exporting a type from here, can you move this to regular dependencies 🙏